portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,15 +35,29 @@
   private static final String TASK_NAME = "Comments";
   private static final String TIMESTAMP = "timestamp";
   private static final String COMMENT_PROPERTY = "comment";
+  private static final String FILTER_PARAM = "filterCount";
```

Comment:
Also, the common terminology for this would be "page_size". And there is a concept of pagination tokens that is out of scope for this, but worth looking into! Here is some reading on it - https://cloud.google.com/apis/design/design_patterns#list_pagination